integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/DistroXUpgradeTests.java

```diff
@@ -42,18 +42,21 @@ protected void setupTest(TestContext testContext) {
 
     @Test(dataProvider = TEST_CONTEXT)
     @UseSpotInstances
-    @Description(given = "there is a running Cloudbreak, and an environment with SDX and DistroX cluster in available state",
-            when = "upgrade called on the DistroX cluster", then = "DistroX upgrade should be successful, the cluster should be up and running")
+    @Description(given = "there is a running Cloudbreak, and an environment with SDX and DistroX cluster in available state", when = "upgrade called on the DistroX cluster", then = "DistroX upgrade should be successful, the cluster should be up and running")
```

Comment:
Please use the standardised format for the test description, like at [DistroXScaleTest(https://github.com/hortonworks/cloudbreak/blob/master/integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/DistroXScaleTest.java#L39-L42): 
```
@Description(
            given = "there is a running cloudbreak",
            when = "a valid DistroX create request is sent",
            then = "DistroX cluster can be resiliently scaled up and down with higher node count")
```